proposals/custom-descriptors/Overview.md

```diff
@@ -592,18 +592,18 @@ declentry       ::= protoconfig descconfig
 
 protoconfig     ::= v:vec(descindex) (if |v| <= 1)
 
-descconfig      ::= exportnames vec(methodconfig)
+descconfig      ::= exportnames:vec(name) methods:vec(methodconfig)
 
-exportnames     ::= vec(name)
+methodconfig    ::= 0x00 methodnames => method
+                  | 0x01 methodnames => getter
+                  | 0x02 methodnames => setter
+                  | 0x03 methodnames vec(methodnames) => constructor
 
-methodconfig    ::= kind:methodkind
-                    methodname:name
-                    exportname:name
+staticmethods   ::= hasinstance:vec(name) methods:vec(methodnames)
```

Comment:
You probably meant to use `staticmethods` on line 600 somewhere.

But I don't think we need it at all: `instanceof` should Just Work™ when we set up the constructor function's `.prototype` property (as the text below already describes). So line 600 as currently written should be enough.

Reply:
Ah good point. I was thinking that users would implement it with a custom function that would do a simple `ref.test_desc`. Do you think there would be any performance benefit from overriding `instanceof` that way, or is the prototype chain check already fast?

Reply:
I haven't measured it, but I'd guess that using `Symbol.hasInstance` is slower than the regular alternative. (That tends to be a general rule of thumb in JS: you _can_ override all sorts of behavior, but you tend to fall off the optimized fast path if you do.)

Reply:
Sounds good. I'll leave this out for now and we can always add it later if some compelling use case comes up.